internal/rbd/controllerserver.go

```diff
@@ -786,16 +787,15 @@ func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 
 	rbdVol, err := genVolFromVolID(ctx, volID, cr, req.GetSecrets())
 	if err != nil {
-		if _, ok := err.(ErrImageNotFound); ok {
+		switch err.(type) {
+		case ErrImageNotFound:
 			return nil, status.Errorf(codes.NotFound, "volume ID %s not found", volID)
```

Comment:
@nixpanic can you avoid return inside each case here? same as you did above

Reply:
sure!